src/app/clusters/audio-output-server/audio-output-server.cpp

```diff
@@ -186,9 +186,18 @@ bool emberAfAudioOutputClusterRenameOutputCallback(app::CommandHandler * command
         emberAfSendImmediateDefaultResponse(EMBER_ZCL_STATUS_FAILURE);
     }
 
-    bool success         = delegate->HandleRenameOutput(index, name);
-    EmberAfStatus status = success ? EMBER_ZCL_STATUS_SUCCESS : EMBER_ZCL_STATUS_FAILURE;
-    emberAfSendImmediateDefaultResponse(status);
+    if (HasFeature(endpoint, AudioOutputFeature::kNameUpdates))
+    {
+        bool success         = delegate->HandleRenameOutput(index, name);
+        EmberAfStatus status = success ? EMBER_ZCL_STATUS_SUCCESS : EMBER_ZCL_STATUS_FAILURE;
+        emberAfSendImmediateDefaultResponse(status);
```

Comment:
Please send status responses via the CommandHandler instead of using the global `emberAfSendImmediateDefaultResponse` (which we are trying to remove).

Reply:
Updated.